test/list/ListProxyTest.js

```diff
@@ -38,7 +38,9 @@ describe("List Proxy", function () {
     });
 
     beforeEach(function () {
-        listInstance = client.getList('test')
+        return client.getList('test').then(function (list) {
```

Comment:
Does this guarantee that listInstance is set or should we call some sort of wait on promise before returning.

Reply:
Yes.